__tests__/integration.test.ts

```diff
@@ -95,19 +98,18 @@ describe('Node host', () => {
     const childHost = cp.spawn(
       process.execPath,
       [path.join(__dirname, '..', 'bin', 'cli.js')],
-      { env: { NVIM_NODE_HOST_DEBUG: 1 } }
+      { env: { NVIM_NODE_HOST_DEBUG: 1 }, stdio: 'ignore' }
     );
-    return axios.get('http://127.0.0.1:9229/json/list').then(
-      res => {
-        childHost.kill();
-        const debuggerData = res.data[0];
-        expect(debuggerData.webSocketDebuggerUrl).toMatch(
-          'ws://127.0.0.1:9229'
-        );
-      },
-      () => {
-        childHost.kill();
+    await sleep(1000);
+    const debugData = await axios.get('http://127.0.0.1:9229/json/list').then(
+      res =>
+        Array.isArray(res.data) && res.data.length ? res.data[0] : undefined,
+      err => {
+        throw err;
```

Comment:
Not sure how `async/await` works here so I threw error from axios just to be sure. Reject callback should return undefined here so that the following `expect` assertion will catch it if throw doesn't work.